test/legacy-component-aliases.test.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * Smoke test that verifies that legacy component names are still exported from
+ * legacy packages to prevent breaking changes while refactoring
+ * package / element names
+ * Background:
+ * - https://github.com/vaadin/web-components/issues/1992
+ * - https://github.com/vaadin/web-components/issues/1993
+ *
+ * TODO: Remove after V23
```

Comment:
Assuming we keep the deprecated symbols until the next LTS at least.

Reply:
Yes, the plan is to remove the deprecated classes in V24.